core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/controllers/WizardRelationalDatasourceController.java

```diff
@@ -131,8 +133,9 @@ public void init( final DatasourceModel datasourceModel ) {
     previewResultsTable = (XulTree) document.getElementById( "previewResultsTable" ); //$NON-NLS-1$
     previewResultsTreeCols = (XulTreeCols) document.getElementById( "previewResultsTreeCols" ); //$NON-NLS-1$
     previewLimit = (XulTextbox) document.getElementById( "previewLimit" ); //$NON-NLS-1$
-    editConnectionButton = (XulButton) document.getElementById( "editConnection" ); //$NON-NLS-1$
-    removeConnectionButton = (XulButton) document.getElementById( "removeConnection" ); //$NON-NLS-1$
+    editConnectionButton = (XulToolbarbutton) document.getElementById( "editConnection" ); //$NON-NLS-1$
+    removeConnectionButton = (XulToolbarbutton) document.getElementById( "removeConnection" ); //$NON-NLS-1$
+    addConnectionButton = (XulToolbarbutton) document.getElementById( "addConnection" ); //$NON-NLS-1$
```

Comment:
Why is this new class property, `addConnectionButton`, needed?

Reply:
In `main_wizard_panel.xul`, I converted 3 buttons to toolbar buttons. In `WizardRelationalDatasourceController.java`, I could find only 2 of them , so I added `addConnectionButton`. Turns out the addition was not needed.